.github/workflows/main.yml

```diff
@@ -47,9 +35,8 @@ jobs:
         ember-try-scenario:
           - ember-lts-3.16
           - ember-lts-3.20
-          - ember-release
-          - ember-beta
-          - ember-canary
+          - ember-lts-3.24
+          - ember-lts-3.28
```

Comment:
Removing bleeding edge and latest [release](https://emberjs.com/releases/release/) channel since this addon definitely doesn't work in ember 5.x